pkg/mcs/resource_manager/server/install/install.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
```

Comment:
install sounds strange.

Reply:
I suggest every service have it's own install package, and there a control panel to import them.

Reply:
`registry/register.go` make more sense to me.

Reply:
there already has a `registry` package, which provide the registry in global view.

Reply:
I see kubernetes also call `install` , ref https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/rbac/install/install.go